wordpress/wp-content/plugins/cds-base/classes/Modules/Contact/ContactForm.php

```diff
@@ -42,45 +42,147 @@ public function render($atts, $content = null): string
+                        $heading = __('Get technical support', 'cds-snc');
+                        $label = __('Describe the problem.', 'cds-snc');
                         break;
                     case 'give-feedback':
-                        $heading = __("Give feedback", "cds-snc");
-                        $label = __("How can we do better?", "cds-snc");
+                        $heading = __('Give feedback', 'cds-snc');
+                        $label = __('How can we do better?', 'cds-snc');
                         break;
                     case 'set-up-a-demo-to-learn-more-about-GC-Articles':
-                        $heading = __("Set up a demo to learn more about GC Articles", "cds-snc");
-                        $label = __("Your contact information", "cds-snc");
+                        $heading = __(
+                            'Set up a demo to learn more about GC Articles',
+                            'cds-snc',
+                        );
+                        $label = __('Your contact information', 'cds-snc');
                         break;
                     case 'other':
-                        $heading = __("Tell us more", "cds-snc");
-                        $label = __("Your message", "cds-snc");
+                        $heading = __('Tell us more', 'cds-snc');
+                        $label = __('Your message', 'cds-snc');
                         break;
                 }
                 ?>
+
+                <?php if ($contactType === 'request-site') : ?>
+                    <form id="contact-form" method="POST" action="/wp-json/contact/v1/process">
+                    <input type="hidden" name="contact-type" value="<?php echo $contactType; ?>"/> 
+                    <h2><?php echo $heading; ?></h2>
+                    <?php wp_nonce_field(
+                        'contact_form_nonce_action',
+                        'contact',
+                    ); ?>
+
+
+                    <div class="focus-group">
+                        <label class="gc-label required" for="email" id="email-label">
+                            <?php _e('Email', 'cds-snc'); ?>
+                        </label>
+                        <input type="email" class="gc-input-text" id="email" required autocomplete="email"
+                               name="email" value="">
+                    </div>
+
+                    <!-- collection name  -->
+                    <div class="focus-group">
+                    <label class="gc-label required" id="collection-name-label" for="gc-collection-name">
+                        <?php _e("What do you want to name your GC Articles collection?", "cds-snc"); ?>
+                    </label>
+                    <input
+                            type="text"
+                            class="gc-input-text"
+                            id="gc-collection-name"
+                            required
+                            aria-describedby="collection-name-label"
```

Comment:
I'm pretty sure you don't need `aria-describedby` when we already have the label pointing to this.